src/js/static.js

```diff
@@ -22,8 +22,8 @@ gpii.express.router["static"].init = function (that) {
     that.staticHandler = express["static"](that.options.content);
 };
 
-gpii.express.router["static"].route = function (that, req, res) {
-    that.staticHandler(req, res);
+gpii.express.router["static"].route = function (that, req, res, next) {
+    that.staticHandler(req, res, next);
```

Comment:
I used `master` to test a theory, and discovered that the failure was consistent if I loaded the static middleware first in the chain.  I made the same change to this branch and confirmed that it still works.   This should catch regressions on this problem.